app/components/shop-sidebar/filters/price-filter/price-filter.tsx

```
@@ -1,9 +1,33 @@
+    params.set("maxPrice", priceValue[1].toString());
+    return `/shop?${params.toString()}`;
+  }, [priceValue, searchParams]);
+
+  const handleClickButton = () => {
+    router.push(url);
+  };
+
+  // Input change handler with basic validation
+  const handlePriceInputChange = (index: 0 | 1, value: string) => {
+    const newValue = Number(value);
+    if (!isNaN(newValue) && newValue >= 0) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)